# This patch enables ciphers in 32b openssl when FIPS is enabled
# Currently not used by OI but if we ever want to go down the FIPS verison
# route then we will need this.
--- openssl-3.0.8/crypto/conf/conf_api.c	2023-02-07 13:43:33.000000000 +0000
+++ openssl-3.0.8-patch/crypto/conf/conf_api.c	2023-04-21 17:22:25.979648496 +0100
@@ -86,6 +86,13 @@
             if (p != NULL)
                 return p;
         }
+	if ((strcmp(section, "_SYS") == 0) && (strcmp(name, "ISA") == 0)) {
+#ifdef _LP64
+                return "64";
+#else
+                return "32";
+#endif /* _LP64 */
+	}
     }
     vv.section = "default";
     vv.name = (char *)name;
